spec/src/main/asciidoc/license-efsl.adoc

```diff
@@ -0,0 +1,76 @@
+[subs="normal"]
+....
+Specification: {doctitle}
+
+Version: {revnumber}
+
+Status: {revremark}
+
+Release: {revdate}
+....
+
+== Copyright
+
+Copyright (c) {inceptionYear}, {currentYear} Eclipse Foundation.
```

Comment:
Have you tested to see whether this works? I thought I tried but it did not work.

Reply:
I did and it works. But you need to set the variables in the `asciidoctor` plugin:

https://github.com/radcortez/microprofile-open-api/blob/4474a6b1816a2b4ecfb38fe1e81b0b66a4bf2f00/spec/pom.xml#L82-L86